_pages/privacy.md

```diff
@@ -2,10 +2,22 @@
 title: Privacy
 ---
 
-See the following sub-sections:
+<p>The GSA Privacy Office preserves and evolves privacy protections for any individual whose personal information is handled by the agency. In practice, this requires the Office to conduct a thorough, periodic analysis of each of the GSA&rsquo;s programs and information systems.</p>
 
-* [Privacy Impact Assessment (PIA)](pia/)
-* [Personally Identifiable Information (PII)](pii/)
-* [System of Records Notice (SORN)](sorn/)
+<p>As someone interested in shipping a product or service, you&rsquo;ll need to first consider <strong>the agency</strong> at which the product or service will be authorized to operate. If that agency is someone <i>other</i> than GSA, please stop reading this guide and contact the Privacy Office at the authorizing agency. Otherwise, take the following steps:</p>
 
-For help, reach out to [#wg-privacy](https://gsa-tts.slack.com/messages/C4F2HMGBE/).
+<ol>
+  <li>
+    <a href="https://gsa.gov/portal/getMediaData?mediaId=199855">GSA Policy 1878.2A CIO P Conducting Privacy Impact Assessments (PIAs)</a> requires that each of the GSA&rsquo;s information systems, either planned or under development, be subject to an annual privacy threshold analysis. Please <a href="https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLScrhrrzgS-ZcOpS37YzLdmVXkwX_3dBefZ1t39oKcgdVepmuQ/viewform">fill out and submit this Google Form</a> to get started.
+  </li>
+  <li>
+    <p>Once they&rsquo;ve had a chance to review your information system, GSA&rsquo;s Privacy Office will reach out to help you collaboratively identify, discuss, and evaluate privacy risks, and to determine the ways in which they might be mitigated. Generally speaking, a system&rsquo;s privacy risk stems from to the degree to which that system and its associated business processes collect and use personally identifiable information (PII).</p>
```

Comment:
Mind moving this second sentence out of the list, to keep the list focused on the review process? Could similarly slim down the other bullet points as well.